src/CMakeLists.txt

```diff
@@ -1,10 +1,20 @@
-# Copyright (c) 2023 The Bitcoin Core developers
+# Copyright (c) 2024-present The Bitcoin Core developers
 # Distributed under the MIT software license, see the accompanying
-# file COPYING or http://www.opensource.org/licenses/mit-license.php.
+# file COPYING or https://opensource.org/license/mit/.
 
 include(GNUInstallDirs)
 include(AddWindowsResources)
 
+option(SECP256K1_DISABLE_SHARED "" ON)
+option(SECP256K1_ENABLE_MODULE_ECDH "" OFF)
+option(SECP256K1_ENABLE_MODULE_RECOVERY "" ON)
+option(SECP256K1_BUILD_BENCHMARK "" OFF)
+option(SECP256K1_BUILD_TESTS "" OFF)
+option(SECP256K1_BUILD_EXHAUSTIVE_TESTS "" OFF)
+option(SECP256K1_BUILD_CTIME_TESTS "" OFF)
```

Comment:
These cache variables might be disabled upstream when building as a subproject.

Reply:
Irrelevant since https://github.com/hebasto/bitcoin/pull/192#discussion_r1594428809.